readme.md

```diff
@@ -40,6 +40,32 @@ console.log(obj);
 //=> {foo: {bar: 'b', baz: 'x', 'dot.dot': 'unicorn'}}
 ```
 
+
+## API
+
+### get(obj, path)
+
+### set(obj, path, value)
+
+#### obj
+
+Type: `object`
+
+Object to get or set the path value.
```

Comment:
I think the current description is fine. Maybe add \` around it so it's clear the path is the argument shown above.